src/crypto/crypto_keygen.cc

```diff
@@ -65,8 +65,7 @@ Maybe<bool> SecretKeyGenTraits::AdditionalConfig(
     SecretKeyGenConfig* params) {
   Environment* env = Environment::GetCurrent(args);
   CHECK(args[*offset]->IsUint32());
-  params->length = static_cast<size_t>(
-      std::trunc(args[*offset].As<Uint32>()->Value() / CHAR_BIT));
+  params->length = args[*offset].As<Uint32>()->Value() / CHAR_BIT;
```

Comment:
Aside: the comment in src/crypto/crypto_keygen.h is wrong, it says it's the number of bits, not bytes.

Reply:
Thanks, I've decided not to change that here but I've opened https://github.com/nodejs/node/pull/44470 to change nits in this and another comment.